loewexy/gpsd-fake
# Prepare for NPM release

Here we go, that's what I did:

* Added [eslint](http://eslint.org) with auto generated config via `eslint --init`. Helped me a lot to match your code style. Will help others to contribute, also.
* Extended .gitignore via <http://gitignore.io>. Just as preparation for other contributors. I added netbeans since u use it.
* Wrapped your main.js in a simple module with a single exported function. That is the biggest diff since I had to indent by one. This allows us to use gpsd-fake as library.
* Added a binary js file in `./bin/gpsd-fake` which uses [yargs](https://github.com/yargs/yargs) to handle CLI options. Also gives us a nice help screen.
* Adjusted the readme

How to test?
1. Clone this branch
2. install via `npm install`.
3. make sure you linked your global npm modules to your $PATH variable. Otherwise add `export PATH="$(npm bin -g):$PATH"` to your `.bashrc` file
4. Do a `npm link` in your repo to link the repo into the global node modules
5. Enjoy the new version via `gpsd-fake` :)

As soon it is published to npm, it will also be available via `gpsd-fake` globally in your system after an `npm install -g gpsd-fake`

Fixes #1 

Let me know what you think,
Benedikt

<img width="989" alt="screen shot 2017-08-07 at 14 18 17" src="https://user-images.githubusercontent.com/1737026/29026308-64efa24c-7b7b-11e7-901d-fe2058136adf.png">
